marklogic-data-hub-central/ui/src/components/entities/mapping/mapping-step-detail/mapping-step-detail.tsx

```diff
@@ -1504,7 +1503,7 @@ const MappingStepDetail: React.FC = () => {
                     </HCCard>
                   </div>
                   :
-                  (interceptorExecuted && interceptorExecutionError !== "") ?
+                  (!interceptorExecuted && interceptorExecutionError !== "") ?
```

Comment:
There's a UI test failure that I think may be caused by the change in logic here. Was it intended to **not** the "interceptorExecuted" here? I think it may need to be the other way around.

Reply:
Oops, yes, I changed the flag just to test locally and forgot to revert the change.